lib/keeper/datastore.rb

```diff
@@ -2,25 +2,37 @@ module Keeper
   module Datastore
     class << self
       # @!visibility private
-      def expire(token, time)
-        set_with_expiry(token, time)
+      def rotate(jti, seconds)
```

Comment:
i'm fine with changing the wording of `token` to `jti` what does the `i` stand for `jwt` idk they're all just letters right now.